rfps/rfp-filecoin-wallets.md

```diff
@@ -0,0 +1,159 @@
+# RFP: Filecoin Wallet & Tools
+
+**Name of Project:** `Filecoin Wallet & Tools`
+
+**Proposal Category:** `app-dev`, `devtools-libraries`
+
```

Comment:
Can you please add yourself as a Proposer here?

Reply:
Can I add @jnthnvctr as well?

Reply:
Yup! Please do!